member/views.py

```diff
@@ -63,6 +71,24 @@ def verify_requestor_is_in_user_band(request, user):
     return True
 
 
+@login_required
```

Comment:
Since these are not really 'views' it'd be better to put them in the helpers file. (I haven't been consistent with this - the 'delete_invite' view isn't really a view either.)